pkg/virt-api/webhooks/s390x.go

```diff
@@ -0,0 +1,64 @@
+/* Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2021
```

Comment:
Please, fix the copyright year

Reply:
done

Reply:
@jschintag I've never seen just

```
Copyright $YEAR
```

I don't think it makes a lot of sense not to list what entities claims the copyright. I think this should be

```
Copyright 2024 IBM, Inc.
```

There's also another file that you add which weirdly claims Red Hat copyright. I think that should be changed too.

Reply:
Well to be fair, i usually copy those arch specific files from another arch to have a template.
So `Copyright 2024 IBM, Inc.` comes from arm64. I think it does not really make sense to have any company in there, since i think in the end the copyright belongs to the kubevirt org.

Reply:
Please follow the same as in this PR: https://github.com/kubevirt/kubevirt/pull/10645

Reply:
So replace
```
Copyright $Year $Owner
```
with
```
Copyright the KubeVirt Authors.
```
?

Reply:
yes

Reply:
done